pipelines/features_extraction/README.md

```diff
@@ -1,3 +1,92 @@
+<img src="https://i.ibb.co/FYNkbbg/extractor-train.png">
+<div align="left">
+
+
+## Validation
+
+Validation part consists of the following steps:
+1. Accumulating all the embeddings in [EmbeddingMetrics](https://open-metric-learning.readthedocs.io/en/latest/contents/metrics.html#embeddingmetrics).
+2. Calculating distances between queries and galleries.
+3. [Optional] Applying some specific retrieval postprocessing [techniques](https://open-metric-learning.readthedocs.io/en/latest/contents/postprocessing.html) like re-ranking.
+4. Calculating retrieval metrics like
+   [CMC@k](https://open-metric-learning.readthedocs.io/en/latest/contents/metrics.html#calc-cmc),
+   [Precision@k](https://open-metric-learning.readthedocs.io/en/latest/contents/metrics.html#calc-precision),
+   [MeanAveragePrecision@k](https://open-metric-learning.readthedocs.io/en/latest/contents/metrics.html#calc-map)
+   or others.
+
+<div align="center">
+<img src="https://i.ibb.co/kcVb7YH/extractor-validation.png">
+<div align="left">
+
+
+## Customization
+
+Pipelines are built around blocks like model, criterion, optimizer and so on.
+Some of them can be replaced by existing entities from OML or by your custom implementations, see the customisation
+[instruction](https://github.com/OML-Team/open-metric-learning/tree/pipeline_readme/pipelines#how-to-use-my-own-implementation-of-loss-model-augmentations-etc).
+
+In feature extraction pipelines you can customize:
+
+|   Name in config   |                                                                                                                                                                    Requirements                                                                                                                                                                    | Where to add my implementation? |                                Where to find the existing implementations?                                |
+|:------------------:|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:-------------------------------:|:---------------------------------------------------------------------------------------------------------:|
+| `transforms_train` |                                                                                                              Callable, see [available](https://github.com/OML-Team/open-metric-learning/tree/pipeline_readme/oml/transforms/images).                                                                                                               |      `TRANSFORMS_REGISTRY`      | [configs](https://github.com/OML-Team/open-metric-learning/tree/pipeline_readme/oml/configs/transforms)   |
```

Comment:
Can it be better to give a link to the registry for transforms? It is easier to see the connection between the parameter in the config and functions.

Or give both links